examples/functions/banner/README.md

```diff
@@ -8,24 +8,26 @@ There are only 3 things to consider when writing a function:
 - Write your output to the standard output
 - If needed, log to the standard error
 
-# Building the `banner-function` image
+# Preparing deployment
+
+## Build the `banner-function` image
 
 Run `make image`, it will create a docker image with the tag `appcelerator/banner-function:latest`
 
-# Local deployment
+## Push your image to a public registry (e.g. DockerHub)
```

Comment:
now that the image push section applies to both cloud and local deployment, it's getting difficult to test locally: not everybody has an account on DockerHub, or wants to push an image that is only for tests. The local registry is there for that reason, we should mention it.